engine/backends/proverif/proverif_backend.ml

```diff
@@ -278,6 +311,27 @@ module Make (Options : OPTS) : MAKE = struct
                           ^^ iblock parens inner_field_doc)
                   in
                   string "Some" ^^ inner_block
+              | PConstruct { name; args }
+              (* Some(expr) -> Some(<expr_typ_to_bitstring>(expr))*)
```

Comment:
Can you rephrase the comment `handles "Some(expr) -> Some..."` or something? I actually thought this was commented code :sweat_smile:

Reply:
Oops, totally! I wrote a more sensible comment explaining what happens and why.